npm/normalize-obstructive-user-agent/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@cypress/normalize-obstructive-user-agent",
+  "version": "0.0.0-development",
+  "description": "Official Cypress plugin to normalize obstructive user agents to appear as common or expected user agents",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc || echo 'built, with type errors'",
+    "build-prod": "yarn build",
+    "check-ts": "tsc --noEmit",
+    "watch": "tsc -w",
+    "cypress:open": "cypress open",
+    "prepublish": "tsc"
+  },
+  "devDependencies": {
+    "cypress": "^10.3.1",
```

Comment:
do we ist cypress as a dev dep with it in the mono-repo?

Reply:
I think it depends. I know the `angular` one does, but its' pinned to `"0.0.0-development"`, which seems more appropriate. Then we can just pin the peer dep to 10+